codegen/allapigen.py

```diff
@@ -59,3 +60,5 @@
     allapigen.generate(version, static_infos)
     t2 = time()
     print(f"Time to generate APIs: {t2 - t1:.2f} seconds")
+    _search("", version="242")
+    _download_nltk_data()
```

Comment:
@mkundu1 @seanpearsonuk 

Download nltk_data during codegen into `generated/nltk_data` folder and package it for pyconsole. 

We have set `NLTK_DATA` environment variable to above location.

Reply:
This is the pyfluent packaging code, so we should remove the above download call for now. We shall provide the data files tothe  build team when we package pyfluent for pyconsole next time.

Reply:
Done. Thank you.